vocabularies/Offline.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Alias="Core" Namespace="Org.OData.Core.V1" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="com.sap.vocabularies.Offline.v1" Alias="Offline">
+      <Annotation Term="Core.Description">
+        <String>Terms for annotating Offline OData</String>
```

Comment:
What is "Offline OData"? Is this an SAP Product? If so, please add a link / short explanation.

Reply:
Good idea,  a long description is added to introduce the product, thanks.